### PR TITLE
Zarrina alexander z

### PR DESCRIPTION
**Pull request by Keldieva**

### CODE REVIEW

File: .github/workflows/action.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        installer-parallel: true
+
+
+    - name: Install poetry dependencies
+      if: steps.cached-poetry-dependencies.outputs.cache-hit != 'true'
+      run: poetry install --no-interaction --no-root
+
+    - name: Run tests
+      run:
+        poetry run pytest -v
```
Comment (akioikeda) [inline .github/workflows/action.yml:41]: My action is ahead of your action

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.idea/
+__pycache__/
+tests/__pycache__/
```
Comment (akioikeda) [inline .gitignore:3]: There is no reason to use line 3, line 2 actually works for all project (including subfolders)

File: pyproject.toml [hunk 3]
```diff
@@ -4,15 +4,15 @@ version = "0.1.0"
 description = ""
 authors = ["akioikeda <principewar@live.com>"]
 readme = "README.md"
-packages = [{include = "saucedemo_05"}]
+#packages = [{include = "saucedemo_05"}]
```
Comment (akioikeda) [inline pyproject.toml:7]: Commented line 7 should be removed from toml file

File: tests/test_demo_kate.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+user_name = 'standard_user'
+password = 'secret_sauce'
+
+
+class TestSample:
+
+    def test_url(self, driver):
+        # login as an authorized user
+        driver.find_element(By.ID, 'user-name').send_keys(user_name)
```
Comment (akioikeda) [inline tests/test_demo_kate.py:11]: My version is ahead of your code

**Review by akioikeda — CHANGES_REQUESTED** [review]
Code needs to be rewritten.
I left some comments under the specific code changes.
Currently I cannot approve this pull request.
Please review your code and compare it with my main branch

**Comment by Keldieva** [comment]
I am sorry. It is just a dummy code for learning purposes. I have pushed it by mistake to this branch.

**Pull request closed without merging**